components/GymPlan/GymPlan.js

```diff
@@ -49,36 +62,38 @@ export default function GymPlan() {
 
    return (
       <StyledForm onSubmit={handleSubmit}>
-         <AddSectionButton
-            type='button'
-            onClick={() => addSection('Warm-Up')}>
-            add Warm-Up
-         </AddSectionButton>
-         <AddSectionButton
-            type='button'
-            onClick={() => addSection('Squat')}>
-            add Squat
-         </AddSectionButton>
-         <AddSectionButton
-            type='button'
-            onClick={() => addSection('Bench')}>
-            add Bench
-         </AddSectionButton>
-         <AddSectionButton
-            type='button'
-            onClick={() => addSection('Deadlift')}>
-            add Deadlift
-         </AddSectionButton>
-         <AddSectionButton
-            type='button'
-            onClick={() => addSection('Assistant')}>
-            add Assistant
-         </AddSectionButton>
-
+         <CenterButtons>
+            <AddSectionButton
+               type='button'
+               onClick={() => addSection('Warm-Up')}>
+               add Warm-Up
+            </AddSectionButton>
+            <AddSectionButton
+               type='button'
+               onClick={() => addSection('Squat')}>
+               add Squat
+            </AddSectionButton>
+            <AddSectionButton
+               type='button'
+               onClick={() => addSection('Bench')}>
+               add Bench
+            </AddSectionButton>
+            <AddSectionButton
+               type='button'
+               onClick={() => addSection('Deadlift')}>
+               add Deadlift
+            </AddSectionButton>
+            <AddSectionButton
+               type='button'
+               onClick={() => addSection('Assistant')}>
+               add Assistant
+            </AddSectionButton>
+         </CenterButtons>
```

Comment:
Instead of hardcoding all buttons, you could map over an array of the 5 different exercises. This would save a lot of code and you could easily extend the array in the future.